multichain-testing/tools/sleep.ts

```diff
@@ -6,13 +6,17 @@ export const sleep = (ms: number, log: Log = () => {}) =>
     setTimeout(resolve, ms);
   });
 
+type RetryOptions = {
+  maxRetries?: number;
+  retryIntervalMs?: number;
+};
+
 const retryUntilCondition = async <T>(
   operation: () => Promise<T>,
   condition: (result: T) => boolean,
   message: string,
-  maxRetries: number,
-  retryIntervalMs: number,
   log: Log,
+  { maxRetries = 6, retryIntervalMs = 3500 }: RetryOptions = {},
```

Comment:
I suggest moving `log` into `RetryOptions`

and include `setTimeout` as an overrideable default as discussed

Reply:
"allow callers ..." is a feat, no?